products/numpy.md

```diff
@@ -0,0 +1,49 @@
+---
+title: NumPy
+permalink: /numpy
+category: framework
+releasePolicyLink: https://numpy.org/neps/nep-0029-deprecation_policy.html
+versionCommand: python -c "import numpy; print(numpy.__version__)"
+auto:
+-   git: https://github.com/numpy/numpy
+-   regex: ^v(?<major>[0-9]+)\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)$
+releases:
+-   releaseCycle: "1.14"
```

Comment:
This should be be reversed in order, so newer releases are on top (See `CONTRIBUTING.md`, which clarifies this).